scripts/dist.py

```diff
@@ -238,6 +244,9 @@ def build_jdk_11_gradle():
     command = "gradlew"
 
     if platform.system() == "Darwin":
+        # make sure gradlew command is executable
+        os.system("chmod +x " + os.getcwd() + "/" + folder + "/gradlew")
```

Comment:
@onurd86  did u have issue with these? 

@man-zhang @seran : you use Mac, isn't it? ever had such issue?

Reply:
@arcuri82 Here is what happened, the script gave an error stating that gradlew is not executable. This is very easy to fix in command line by chmod +x command. I thought it would be a nice addition to make sure the file is executable before running gradlew command.